src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectState.cs

```diff
@@ -35,37 +36,26 @@ internal class ProjectState
     private readonly object _lock;
 
     private readonly IProjectEngineFactoryProvider _projectEngineFactoryProvider;
+    private readonly LanguageServerFeatureOptions _languageServerFeatureOptions;
     private RazorProjectEngine? _projectEngine;
 
     public static ProjectState Create(
```

Comment:
❓ I've always wondered why we have a create here that doesn't do any work other than just call the constructor with the same things. Not unique to this PR though

Reply:
💭 I suspect it's modeled on Roslyn's APIs like `Compilation`.